content/id/docs/tutorials/kubernetes-basics/deploy-app/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: _Deploy_ Aplikasi
```

Comment:
Italics in the title will not work. Also, please give some Indonesian words. 
```suggestion
title: Menyebarkan (Deploy) Aplikasi
```

Reply:
how about "Menjalankan"? true that we can't use italic on title
```suggestion
title: Menjalankan Aplikasi
```

Reply:
Other alternative for deploy translation: menggelar
Source: https://twitter.com/ivanlanin/status/1106995180756860928?s=20

Reply:
I doubt our people are familiar with "menyebarkan". But since "menjalankan" is already used by other translation such as "run", I think it's better to use "menyebarkan" instead, with parentheses "(deploy)"